### DISCUSSION TITLE
2007/02/15/%E6%9C%80%E8%BF%91%E6%B2%A1%E6%97%B6%E9%97%B4%E6%9B%B4%E6%96%B0%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#145] [Announcements]
# 2007/02/15/%E6%9C%80%E8%BF%91%E6%B2%A1%E6%97%B6%E9%97%B4%E6%9B%B4%E6%96%B0%E4%BA%86

有些日子没来更新 blog 了，给我的留言也没有处理。最近不论是工作还是私事都特别忙（公司在忙着搬家，我在忙着结婚:），看样子整个二月份都没啥心情写东西了。

https://blog.qizhen.xyz/2007/02/15/%E6%9C%80%E8%BF%91%E6%B2%A1%E6%97%B6%E9%97%B4%E6%9B%B4%E6%96%B0%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

峰.John
2007年04月6日 01:15 编辑
要结婚呀!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
恭喜恭喜~~~~~~~~~~~~~~~~~
呵呵呵呵~!~!~!~!~!~!~!~!~!~!~!!
也感染我下吧!!!嘿嘿

回复
秋
2007年03月23日 10:09 编辑
是很久没有来更新了~

回复
浩
2007年03月17日 21:23 编辑
好久没在你的空间上留言了，一看都要结婚了~！羡慕啊~！
 

回复
竹贤
2007年03月15日 14:21 编辑
我是你blog的常客，读你的文章受益匪浅，很久没有看到你写新文章了，怪想的。千万挤出时间写下去。拜托了。

回复
冰蓝
2007年03月13日 18:36 编辑
那就新婚愉快吧,
呵呵~
期待幸福地归来“

回复
晨恺
2007年03月12日 16:56 编辑
偶也想结婚,可是人丑.没钱.最主要的是没有对象.羡慕啊~~~

回复
patty
2007年03月9日 16:14 编辑
上网在查有关经济学的书籍的读后感——我们的寒假作业，偶然间就连结到你这里来了~近来吓我这一大跳，遇到个这么专业的人！好多看不懂的图喔~太强了“
要结婚了？嘿嘿，恭禧恭禧哦~

回复
Lu
2007年03月8日 15:06 编辑
要结婚啦？恭喜恭喜啊！
以后的日子会越来越幸福哦！

回复
2004
2007年02月27日 11:36 编辑
祝福您，生活事业两丰收！
不过，还要麻烦您的是，我最近开始接触Labview，碰到了一个问题：怎样编写一个进行用户权限管理的vi。已分配权限的用户名和密码已经逐一保存在相应的文件中。在工程项目中，这类问题应该比较常见，您能否帮我找一个类似的小例子，发到我的邮箱中，非常感谢！

回复
洋
2007年02月26日 03:32 编辑
要结婚了啊~~幸福哦~~祝福你们哦~

回复
maggie青雨
2007年02月25日 11:46 编辑
随便走走

回复
周捷
2007年02月25日 10:16 编辑
祝贺婚姻美满幸福, 心想事成!!!

回复
jing
2007年02月24日 20:54 编辑
好久没过来串门了,开始工作后接触的东西远比以前多很多,可写的东西该很多~但却不知为何没以前那种朝花夕拾的定力了,惭愧!总感觉婚姻如此遥远,但却如此的触手可得.
呵呵ANYWAY,祝福你们喔~
 

回复
比禾
2007年02月23日 14:23 编辑
祝婚姻美满!

回复
斌
2007年02月18日 19:05 编辑
恭喜恭喜！ 期待您的文章！ ^ ^ ：）

回复
Wei
2007年02月17日 16:50 编辑
恭喜阿，有情人终成眷属，比翼双飞，连理鸳鸯，。。。

回复
Jing
2007年02月15日 11:15 编辑
祝福！

回复
